membership_delegated_partner/models/membership_line.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MembershipLine(models.Model):
+    _inherit = 'membership.membership_line'
+
+    @api.model
+    def create(self, vals):
+        """ Delegate the member line to the designated partner """
+        if 'account_invoice_line' not in vals:
+            return super(MembershipLine, self).create(vals)
+        line = self.env['account.invoice.line'].browse(
+            vals['account_invoice_line'])
```

Comment:
@chienandalu `vals.get('account_invoice_line', False)`

Reply:
We're going to have it anyway otherwise the method is breaked before